README.md

```diff
@@ -1,43 +1,43 @@
-# Reseau-Social-Libre
+# RSL
 
-> Réseaux social libre est un projet de réseau social non censuré et open source sous licence GPLv3.
+> RSL is a non censured social network open source project under GPLv3 licence.
 > 
-> Tous le monde peut y participer pour:
-> 1. Le modifier.
-> 2. L'améliorer.
-> 3. Apporter ses idées.
+> Everyone can participate to:
+> 1. Modify it.
+> 2. Improve it.
+> 3. Bring his ideas.
 
-## But de ce projet.
+## Goal of this project.
 
-> Ce projet est open source et n'est affilié à aucune organisation ni aucun pays.
+> This project is open source and is not affiliated with any organization or country.
```

Comment:
his project is open source and is not affiliated over any organization or country